apps/api/src/service/application-service.ts

```diff
@@ -46,23 +47,29 @@ const applicationService = (db: PostgresDb) => ({
 				if (!newApplicationRecord[0]) throw new Error('Application record is undefined');
 
 				// Create associated ApplicationContents
-				const { id } = newApplicationRecord[0];
+				const { id: application_id } = newApplicationRecord[0];
 
 				const newAppContents: typeof applicationContents.$inferInsert = {
-					application_id: id,
+					application_id,
 					created_at: new Date(),
 					updated_at: new Date(),
 				};
 				const newAppContentsRecord = await transaction.insert(applicationContents).values(newAppContents).returning();
 				if (!newAppContentsRecord[0]) throw new Error('Application contents record is undefined');
 
+				// Create associated Actions
+				const actionService = initActionService(db);
+
+				const actionResult = await actionService.create(newApplicationRecord[0]);
+				if (!actionResult.success) throw new Error(actionResult.errors);
+
```

Comment:
"init.." says to me this is something static that won't change until we restart the server.
like initialising a config from env vars
vs
some object closure state

looks to me (without knowing full context on this project) that this is a service that will be used a lot.
So with the DI pattern you've got going on here, maybe it would feel better _something_ like:
```js
import {actionService} from 'actionService'; 
const actionResult = await actionService(db).create(newApplicationRecord[0]);
```

Reply:
popped the action service as a named export in that example
the annoying thing about default exports is they make renaming them trivial.

imagine ending up with `initCreateService`, `startActionService`, `actionService`, etc scattered in the project because no one looked to see how it was been imported in other files. if you use a named export, it enforces consistent naming.